package.json

```diff
@@ -206,7 +206,7 @@
     "@elastic/eslint-plugin-kibana-custom": "1.0.3",
     "angular-mocks": "1.4.7",
     "babel-eslint": "7.2.3",
-    "bemify": "0.1.4",
+    "bemify": "0.1.2",
```

Comment:
I had to downgrade because it doesn't look like 0.1.4 exists yet: https://www.npmjs.com/package/bemify